src/Contacts/CNEnums.cs

```diff
@@ -85,6 +85,7 @@ public enum CNErrorCode : nint {
 		CommunicationError = 1,
 		DataAccessError = 2,
 		AuthorizationDenied = 100,
+		NoAccessableWritableContainers = 101,
```

Comment:
Did not realize it wasn't "accessible" but instead "accessable". Thanks `ApiTypoTest`.

Anyway, "accessable" isn't a word. It's likely a typo the person adding the value did. We shouldn't copy Apple on this one, please go for `NoAccessibleWritableContainers`.

Reply:
hahaha OMG, I should have put more attention to the definition xD

> Misspelling of accessible.

Thanks @VincentDondain